repour/adjust/gradle_provider.py

```diff
@@ -129,46 +140,25 @@ async def get_result_data(work_dir, extra_parameters, results_file=None):
 
         """
 
-        template = {
-            "VersioningState": {
-                "executionRootModified": {
-                    "groupId": None,
-                    "artifactId": None,
-                    "version": None,
-                }
-            },
-            "RemovedRepositories": [],
-        }
-
-        manipulation_file_path = os.path.join(work_dir, MANIPULATION_FILE_NAME)
-
-        logger.info(
-            "Reading '{}' file with alignment result".format(manipulation_file_path)
-        )
-
-        if os.path.exists(manipulation_file_path):
-
-            with open(manipulation_file_path, "r") as f:
-                result = json.load(f)
-                template["VersioningState"]["executionRootModified"][
-                    "groupId"
-                ] = result["group"]
-                template["VersioningState"]["executionRootModified"][
-                    "artifactId"
-                ] = result["name"]
-                template["VersioningState"]["executionRootModified"][
-                    "version"
-                ] = result["version"]
-
-            try:
-                template["RemovedRepositories"] = util.get_removed_repos(
-                    work_dir, default_parameters
-                )
-            except FileNotFoundError as e:
-                logger.error("File for removed repositories could not be found")
-                logger.error(str(e))
-
-        return template
+        alignment_report_file_path = os.path.join(work_dir, ALIGNMENT_REPORT_FILE_NAME)
+        manipulation_file_path = os.path.join(work_dir, GME_MANIPULATION_FILE_NAME)
+
+        if os.path.isfile(alignment_report_file_path):
+            file_path = alignment_report_file_path
+            logger.info("Reading '{}' file with alignment result".format(file_path))
+            return pme_provider.parse_pme_result_manipulation_format(
+                work_dir,
+                default_parameters,
+                open(file_path).read(),
+                group_id,
+                artifact_id,
+            )
+        else:
```

Comment:
Am I understanding this flow correctly - for _either_ GME or PME, if `alignmentReport.json` exists it uses `pme_provider.parse_pme_result_manipulation_format` (even for GME) ? Otherwise it falls back to the old `manipulation.json` - which only exists for GME and is a different schema?

What happens if `manipulation.json` exists but is the **report** schema type (i.e. from PME) ? 

Unless, for PME, you always passed `-DreportJSONOutputFile=target/alignmentReport.json` to ensure its a consistent name, regardless of the version of PME ? (which could be removed once PME 4.3 is in production) ?

Reply:
Perfect! Thanks for the clarification.